skbio/util/_testing.py

```diff
@@ -8,6 +8,53 @@
 
 import os
 import inspect
+from nose import core
+from nose.tools import nottest
+from future.utils import PY3
+
+
+@nottest
+class TestRunner(object):
+    """Simple wrapper class around nosetests functionality.
+
+    Parameters
+    ----------
+    filename : __file__ attribute passed in from the caller. This tells the
```

Comment:
The correct docstring format is to have the type listed after the colon, and the description on an indented line below that. Can you please update?